server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -222,6 +242,10 @@ public DefaultConnectOptionsContext(ServiceConnectOptionsType connectOptions, Se
         }
 
         this.sslEncryptionEnabled = (sslEncryptionEnabled == null) ? true : sslEncryptionEnabled;
```

Comment:
I am using tmp variable so that the class fields can be final. I don't like fields getting changed multiple times and that's what happening to other fields. I will wait for kris's changes.